Game elements/game.js

```diff
@@ -14,18 +33,86 @@ class Food {
           this.positionY = Math.random() * 80;
           this.food.style.bottom = this.positionY + 2 +"%";
           this.positionY;
+
      }
 
-     runToLeft() {
-          setInterval(() => {
-               if (this.positionX >= 95) {
-                    this.food.remove();
-               }
-               if (this.positionX <= 100) {
-                    this.food.style.left = this.positionX + "%";
-                    this.positionX += .4;
-               }
-          }, 50);
+     runToRight() {
+
+          this.runningInterval = 
+               setInterval(() => {
+                    if (this.positionX >= 95) {
+                         
+                         
+
+                         let foodRect = this.food.getBoundingClientRect();
+                         let humanRect = human.human.getBoundingClientRect();
+
+                         if(foodRect.top > humanRect.top 
+                              && foodRect.bottom < humanRect.bottom)
+                         {
+                              // console.log("Eaten: "+this.foodType);
+                              // // food was eaten
+
+                              if (this.foodType == 'good') {
+                                   //dodawanie punktow
+                              } else if(this.foodType == 'bad') {
+                                   // odejmuje zycie
```

Comment:
komentarze, pisz taki kod, zeby nie musiec pisac komentarzy, czyli uzuwaj funkcji. Dobra nazwa funkcji odda Ci to co ten kod robi